modules/client/examples/02-multisig-client/02-create-proposal.ts

```diff
@@ -47,6 +47,7 @@ const metadataUri: string = await multisigClient.methods.pinMetadata(
 const proposalParams: CreateMultisigProposalParams = {
   pluginAddress: "0x1234567890123456789012345678901234567890",
   metadataUri,
+  failSafeActions: [],
```

Comment:
is this _allowFailureMap?
can you add some comment explaining what it is ?

Reply:
+1

Reply:
yes, its the _allowFailureMap but as an array of booleans